### DISCUSSION TITLE
Auto zoom Speed (reverse) + Distance not working correctly

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#252] [Q&A] [answered]
Auto Zoom Revers works very well but in combination with the distance (5th option) it doesn't work as expected. In circles and cities it zooms in again independent from the speed what is ok. But when you drive fast, it doesn't zoom in, but zooms out like in the third action. I had recognized this recently when driving by car.

**Comment by devemux86** [comment]
@DolomitesTom thanks for the report.

Distance auto zoom always zoom in when we approach the turn instructions.
That's why the title is "Speed (reverse) + Distance", only the speed is reverse.

How the distance can be reverse?

**Reply by devemux86** [reply]
Distance auto zoom works by keeping the next turn instruction inside the screen.
As we approach a turn instruction, there is available space to zoom in more.

(limited by selected minimum / maximum auto zoom settings)

**Reply by WolfgangZ42** [reply]
I tried "speed + distance" (of course not "reverse") and it has confused me - so I stick with "speed" and I'm happy!

**Reply by devemux86** [reply, marked as answer]
@WolfgangZ42

> I tried "speed + distance" (of course not "reverse") and it has confused me - so I stick with "speed" and I'm happy!

"Speed + Distance" auto zoom works roughly like this:
- Away from turn points: high speed > zoom out, low speed > zoom in
- Near the turn points:  zoom in as we approach the next turn

This is a simple explanation, smooth filters are also involved in the algorithms!

**Comment by DolomitesTom** [comment]
> How the distance can be reverse?

I didn't say that distance can be reverse. I only mean the speed reverse part in this combination mode. Normally when driving fast it should zoom in, but it does the other way round or the distance part of this combination mode overrules this speed reverse part. Don't know how should I express this.

**Reply by devemux86** [reply]
Probably the combination "Speed (reverse) + Distance" has no real value and should not exist as option?

Because they fight each other, better just use the "Speed (reverse)" if want the reverse zoom behavior?

**Comment by DolomitesTom** [comment]
> Probably the combination "Speed (reverse) + Distance" has no real value and should not exist as option?

Could be, but let me test a bit more. Don't delete it. I guess I am the only one who use this option. But I like the feature of zooming in on circles. So then I guess I need to live with the other disadvantage 😁.

**Comment by Cer-o** [comment]
I also have auto zoom set to Speed + Distance.

**Comment by DolomitesTom** [comment]
The beauty is that everyone can find the option that suits them and everyone is happy. In other apps you don't even have the possibility to influence the zoom behaviour. There's nothing like Cruiser.
